File: Guanabara - PHP/exercicios-PHPOO/Aula12-Polimorfismo/Cachorro.php
<?php
include_once 'Lobo.php';
class Cachorro extends Lobo
{
    //Métodos Especiais
    function enterrarOsso()
    {
        echo "<p>Enterrando osso...</p>";
    }
    function abanarRabo()
    {
        echo '<p>Abanando o rabo</p>';
    }
    //Método implementado usando polimorfismo de sobreposição
    public function emitirSom() {
        echo '<p>Cachorro latindo: Au!Au!Au!</p>';
    }
}

File: Java/Meus exercícios POO/Aula12.Polimorfismo/src/main/java/polimorfismo/Cachorro.java
package polimorfismo;
public class Cachorro extends Mamifero {
    //Métodos Públicos
    public void enterrarOsso() {
        System.out.println("Enterrando osso...");
    }
    
    public void abanarRabo() {
        System.out.println("Abanando rabo...");
    }
    
    //(Polimorfismo de Sobreposição)
    @Override
    public void emitirSom() {
        System.out.println("Latindo Au Au Au");
    }
}
